src/handlers/createFile.ts

```diff
@@ -0,0 +1,43 @@
+import * as fs from 'fs';
+import * as mkdirp from 'mkdirp';
+import * as path from 'path';
+import * as ejs from 'ejs';
+import {isUndefined} from 'lodash';
```

Comment:
please use !isDefined and remove loadash, please make sure { always space around }